sdk/search/Azure.Search.Documents/src/Indexes/Models/SearchIndexer.cs

```diff
@@ -8,7 +8,7 @@ namespace Azure.Search.Documents.Indexes.Models
 {
     public partial class SearchIndexer
     {
-        [CodeGenMember("etag")]
+        [CodeGenMember("ETag")]
```

Comment:
@AlexanderSher,  I believe the new customization is now case senstive for comarison.  Please help to confirm whether this is expected.  Thanks

Reply:
Yes, it is expected. Member name should match exactly, otherwise we may get some unexpected results (e.g. private field that can be used in customized code is removed along with its public property wrapper).